templates/rabbitmq.config.epp

```diff
@@ -25,6 +25,21 @@
     {cluster_nodes, {[], <%= $rabbitmq::config::cluster_node_type %>}},
     <%- } -%>
     {cluster_partition_handling, <%= $rabbitmq::config::cluster_partition_handling %>},
+<% } -%>
+<% if $rabbitmq::config::quorum_membership_reconciliation_enabled {-%>
```

Comment:
so it's playing Ok here with either undef or true / false, even with the type still optional on this?

Optional, non-blocking feedback, but would it be possible to indent things (the templating, not the actual config) in a way that makes the nested conditionals a little more obvious? Sort of like [here](https://github.com/voxpupuli/puppet-rabbitmq/blob/acf508dee801eec0ddb5d5d09873f81e13826a78/templates/rabbitmq.config.epp#L21-L28)?